app/javascript/tests/public_health/PatientsTable.test.js

```diff
@@ -0,0 +1,152 @@
+import AdvancedFilter from '../../components/public_health/query/AdvancedFilter.js'
+import CustomTable from '../../components/layout/CustomTable'
+import CloseRecords from '../../components/public_health/actions/CloseRecords';
+import UpdateCaseStatus from '../../components/public_health/actions/UpdateCaseStatus';
+import UpdateAssignedUser from '../../components/public_health/actions/UpdateAssignedUser';
+import InfoTooltip from '../../components/util/InfoTooltip';
+import { mockJurisdiction1, mockJurisdictionPaths } from '../mocks/mockJurisdiction'
+import { mockExposureTabs, mockIsolationTabs } from '../mocks/mockTabs'
+import { mockMonitoringReasons } from '../mocks/mockMonitoringReasons'
+
+const authyToken = "Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==";
+const setQueryMock = jest.fn();
+const setMonitoreeCountMock = jest.fn();
+const dropdownOptions = [ 'Close Records', 'Update Case Status', 'Update Assigned User' ];
+
+function getExposureWrapper() {
+  return shallow(<PatientsTable authenticity_token={authyToken} jurisdiction_paths={mockJurisdictionPaths} workflow={'exposure'} jurisdiction={mockJurisdiction1}
+    tabs={mockExposureTabs} monitoring_reasons={mockMonitoringReasons} setQuery={setQueryMock} setFilteredMonitoreesCount={setMonitoreeCountMock}/>);
+}
+
+function getIsolationWrapper() {
+  return shallow(<PatientsTable authenticity_token={authyToken} jurisdiction_paths={mockJurisdictionPaths} workflow={'isolation'} jurisdiction={mockJurisdiction1}
+    tabs={mockIsolationTabs} monitoring_reasons={mockMonitoringReasons} setQuery={setQueryMock} setFilteredMonitoreesCount={setMonitoreeCountMock}/>);
+}
+
+// Set to exposure workflow
+function getInstance() {
+  return shallow(<PatientsTable authenticity_token={authyToken} jurisdiction_paths={mockJurisdictionPaths} workflow={'exposure'} jurisdiction={mockJurisdiction1}
+    tabs={mockExposureTabs} monitoring_reasons={mockMonitoringReasons} setQuery={setQueryMock} setFilteredMonitoreesCount={setMonitoreeCountMock}/>).instance();
+}
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('PatientsTable', () => {
+  it('Properly renders all main components', () => {
+    const wrapper = getExposureWrapper();
+    expect(wrapper.find('#search').exists()).toBeTruthy();
+    expect(wrapper.find('#tab-description').exists()).toBeTruthy();
+    expect(wrapper.find('#clear-all-filters').exists()).toBeTruthy();
+    expect(wrapper.containsMatchingElement(JurisdictionFilter)).toBeTruthy();
+    expect(wrapper.containsMatchingElement(AssignedUserFilter)).toBeTruthy();
+    expect(wrapper.containsMatchingElement(AdvancedFilter)).toBeTruthy();
+    expect(wrapper.containsMatchingElement(CustomTable)).toBeTruthy();
+    expect(wrapper.containsMatchingElement(DropdownButton)).toBeTruthy();
+    expect(wrapper.find(Dropdown.Item).length).toEqual(3);
+
+    const defaultTab = Object.keys(mockExposureTabs)[0]
```

Comment:
Can you duplicate this tests for the isolation workflow as well?  Anytime the rendering is slightly different it's best to write two

Reply:
Will make this adjustment with my next commit.